src/http/status_code.cr

```diff
@@ -0,0 +1,162 @@
+class HTTP::StatusCode
+  # A support class that provides additional around HTTP status codes.
+  #
+  # Based on [Hypertext Transfer Protocol (HTTP) Status Code Registry](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml)
+  #
+  # It provides constants for the defined HTTP status codes as well as helper
+  # methods to easily identify the type of response.
+
+  CONTINUE                           = 100
```

Comment:
I don't like how there need to be two HTTP status code lists now. The constants here and the `case` in `default_message_for`. It's not nice when you have to maintain two almost identical things. I wish there was a way to have only one list.

Reply:
No, this is fine. The list doesn't change often anyway. It could obviously be dry-ed but that would cost either at runtime or compile time.